otelcollector/configmaps/ama-metrics-settings-configmap.yaml

```diff
@@ -29,6 +29,8 @@ data:
     controlplane-kube-scheduler = false
     controlplane-kube-controller-manager = false
     controlplane-etcd = true
+    acstor-capacity-provisioner = false
```

Comment:
dont we want to default both these new targets to "true" , so thay way, customer doesn't have to go thru additional step to enable thru config map ?

Reply:
But what if a cluster doesn't have ACStor enabled? Would keeping the default to `true` give out any errors when it doesn't find the targets?

Reply:
since these are pod discoveries, if those pods aren't there , it wont discover and hence wont scrape..

Reply:
Makes sense. So, in case if ACStor is installed later, will the customer need to re-apply the config map? Or does the discovery happen automatically?

Reply:
Updated